README.md

```diff
@@ -1,17 +1,19 @@
+Eng | [繁](README.zho.md) | [簡](README.chs.md)
```

Comment:
simplified doesn't exist yet right?

Reply:
that'd mean one of the two has to become a source of truth. Does that sound like a good idea?

Reply:
@Endermanbugzjfc it just feels too complex to setup a workflow just for translating the document.